### ISSUE TITLE
Using PacSum on Windows

### ISSUE THREAD

**Issue by alkakhurana** [mswellhao/PacSum#6]
Hi

Can I clone and run this code on Windows system?

Thanks

**Comment by FrancisYu2020** [comment]
> Hi
> 
> Can I clone and run this code on Windows system?
> 
> Thanks

Yes you can but not simply run in windows cmd. I am using windows and just ran the code. I use WSL Ubuntu as the shell because it seems that the dependency module pyrouge has some nasty setup that probably easier in Linux environment. (Some tutorials can help you out in pure windows but all that I found were in Chinese and are much more complicated than setup in Linux.

All other dependencies are easy to deal with except for pyrouge which for some reason is no longer maintained. I used this tutorial to help me out: https://poojithansl7.wordpress.com/2018/08/04/setting-up-rouge/

Hope this will help!
